package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chpr-logger",
-  "version": "2.7.0",
+  "version": "2.8.0",
```

Comment:
2.8 is already used :) you should use 2.9 instead (and change it [here](https://github.com/transcovo/chpr-logger/blob/master/index.d.ts#L1) too)

Reply:
👍 Yep I forgot tu pull my local branch :p